### DISCUSSION TITLE
数学公式中使用字体\mathcal{}显示效果却是\mathscr{}，如何解决？

### DISCUSSION THREAD

**Discussion by XunXuChinese** [stone-zeng/fduthesis#270] [Q&A] [answered]
我想使用书法艺术体使用了 `\mathcal{}` 命令，但是编译后的显示效果却是书写体（字体具体效果见链接），是 `\mathscr{}` 产生的效果。模板中使用 `\mathcal{}` 或 `\mathscr{}` 编译后的显示效果都是一样的。请问如何解决这个问题？

<https://blog.csdn.net/sdnuwjw/article/details/119920649>

**Comment by stone-zeng** [comment, marked as answer]
使用默认字体（XITS）的话：

```tex
\documentclass{fduthesis}
\setmathfont[range={\mathcal},StylisticSet=1]{XITSMath-Regular.otf}
\begin{document}
\[ \mathcal{F} + \mathscr{F} \]
\end{document}
```

<img width="345" alt="Screenshot 2022-08-24 at 02 33 00" src="https://user-images.githubusercontent.com/18233088/186236890-c2c13670-936d-401a-807a-ac8e9ef8553f.png">

如果要用 Latin Modern 字体（LaTeX 默认风格）：

```tex
\documentclass{fduthesis}
\fdusetup{style/font=lm}
\setmathfont[range={\mathscr},StylisticSet=2]{XITSMath-Regular.otf}  % 这里还是用 XITS 的花体字
\begin{document}
\[ \mathcal{C} + \mathscr{C} \]
\end{document}
```

<img width="345" alt="Screenshot 2022-08-24 at 02 30 49" src="https://user-images.githubusercontent.com/18233088/186237158-ad8d0072-7467-4dac-a00c-a455ccc035cb.png">

（参考 #211）
